lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -453,17 +455,26 @@ type SelectCoins n = "wallets"
 -------------------------------------------------------------------------------}
 
 type Transactions n =
-    CreateTransaction n
+    SignTransaction n
+    :<|> CreateTransactionOld n
     :<|> ListTransactions n
-    :<|> PostTransactionFee n
+    :<|> PostTransactionFeeOld n
     :<|> DeleteTransaction
     :<|> GetTransaction n
 
+-- | https://input-output-hk.github.io/cardano-wallet/api/#operation/postSignTransaction
+type SignTransaction n = "wallets"
+    :> Capture "walletId" (ApiT WalletId)
+    :> "transactions"
+    :> "sign"
+    :> ReqBody '[JSON] PostSignTransactionData
+    :> PostAccepted '[JSON, OctetStream] ApiSerialisedTransaction
```

Comment:
we want both json and octet, not just octet?

Reply:
Is it connected with the fact that we want both binary (serviced by octet) and hex string (serviced by json)?

Reply:
The user can decide whether they want raw transaction bytes, or hex-encoded bytes inside a JSON object.